src/pages/en/guides/deploy/google-cloud.mdx

```diff
@@ -11,21 +11,80 @@ i18nReady: true
 
 Deploying a project to Google Cloud requires some UI clicks. (Most of these actions can also be done using the [gcloud CLI](https://cloud.google.com/sdk/gcloud/)).
 
-### Cloud Run
+### Cloud Run (SSR and static)
+
+Cloud Run is a serverless platform that allows you to run a container without having to manage any infrastructure. It can be used to deploy both static and SSR sites.
+
+#### 1. Preparing the service
 
 1. Create a new GCP project, or select one you already have.
 
 2. Make sure the Cloud Run API is enabled.
 
 3. Create a new service.
 
-4. Use a container from Docker Hub or build your own using [Cloud Build](https://cloud.google.com/build).
+4. Make an empty `Dockerfile` in the root of your project.
```

Comment:
```suggestion
4. Make an empty `Dockerfile` in the root of your project.
```
Can you double-check that it isn't supposed to be `.Dockerfile` with the dot?

Reply:
Yes, it's `Dockerfile`

source: <https://docs.docker.com/engine/reference/builder/>